test/ui-test/lightspeedUiTest.ts

```diff
@@ -590,7 +590,7 @@ export function lightspeedUIAssetsTest(): void {
       const notification = notifications[0];
       expect(await notification.getMessage()).equals(
         "Enable lightspeed services from settings to use the feature.",
-      ).to.be.true;
```

Comment:
This is not directly related to the wizardId support, but was found when running the UI tests for verifying this PR.  I am not sure how it was successfully completed before...